src/spinner/spinner.ts

```diff
@@ -3,7 +3,7 @@ import { property } from 'lit/decorators.js';
 import { classMap } from 'lit/directives/class-map.js';
 
 export class Spinner extends LitElement {
-  @property() variant: 'primary' | 'inverse' = 'primary';
+  @property() variant: 'normal' | 'inverse' | 'inherit' = 'inherit';
```

Comment:
`default` | `primary`| `inverse` are better names for variants. and the default can use the 'currentColor' for the color property.

Reply:
The design system uses "normal" instead of "primary" for spinner variant. I changed it to match the design system.